src/Plugin/client_error/AccessContentReport.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Drupal\client_error_trace\Plugin\client_error;
+
+/**
+ * Report the results of an AccessContent check.
+ *
+ * @see AccessContent
+ *
+ * @class AccessContentReport
+ * @package Drupal\client_error_trace\Plugin\client_error
+ */
+class AccessContentReport extends ReportBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function resultMessage() {
+    switch ($this->result) {
+      case static::SUCCESS:
+        return t("Anonymous users have the 'access content' permission.");
```

Comment:
Maybe you could defer `t()` to a method (`$this->t()`) in `ReportBase`. That way you can stub it to `format_string` or `sprintf` for unit testing if you need to (since `t()` needs bootstrap for the DB to be ready).

A trait would be even cooler (?), so you could use it across multiple classes without a common parent.